src/themes/default/components/button.js

```diff
@@ -102,7 +103,8 @@ const appearances = {
   },
   minimal: {
     backgroundColor: 'transparent',
-    color: (theme, props) => theme.colors[colorKeyForIntent(props.intent)],
+    color: (theme, props) =>
+      props.color || theme.colors[colorKeyForIntent(props.intent)],
```

Comment:
Should the theme be responsible for dictating how prop specificity works? Or should the component let more explicit styles take precedence over theme styles?

Reply:
The alternative would be that the Button component itself would let `props.color` take priority over `themeProps.color`

Reply:
Honestly, I could see it going either way. You may not want to be able to overwrite the color for an appearance or intent. That being said, I think it would be reasonable to expect that any style/box props put on a component should be respected. So it might just be that we need to update useStyleConfig to respect the expected hierarchy

Reply:
thoughts @mshwery ?

Reply:
Will defer to @akleiner2 when he returns. Think we can merge this for now!